### PR TITLE
15custom fonts

### PR DESCRIPTION
**Pull request by tisidro**
# Description
This is a documentation of learning as I work through a [Zastrow CSS Tutorial](link url here). The CSS is not written by me. It may have some revisions and seem to overlap as I walk through the tutorial process. The key focus areas are listed in Validation below. Here is the [Jira Ticket](https://sparkbox.atlassian.net/browse/FSA22V2-60?atlOrigin=eyJpIjoiM2FiNGY4ZThkNDM1NDIzZmE3M2NmMzM3MDczZjU4YjUiLCJwIjoiaiJ9) to give more details and info! :)
## Validation
- [ ]  Code is proofread.
- [x]  Comments are relevant/well written.
- [x]  Documents are formatted.
- [x]  Commit messages are appropriate  and "tell a story" of how the code was written.
Thank you for helping with my progress in Github workflow and CSS mastery! :)

### CODE REVIEW

File: 15custom-fonts/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    - Copy link and place after stylesheet link in HTML head
+    - Add the font name in quotes before the first font in the font-family property of the selector you want to style followed by a comma
+
+- You can also load fonts on to your project directory and access those with the @font-face.
+    - Unzip the file
+    - Drag the folder to your project directory
+    - Write an @font-face rule. Example:  see example in section “Loading a Self-Hosted Font with @font-face” on [this page](https://www.digitalocean.com/community/tutorials/how-to-load-and-use-custom-fonts-with-css). Scroll down and you will see it!
+
+    - Add font name to the selector’s font-family you want to specify in CSS file.
+    - Remember, you don’t need to link anything in the head section under the stylesheet link if you are hosting your own font.
```
Comment (arnest00) [inline 15custom-fonts/README.md:23]: This is repeated on line 29, unless you meant to do that

**Review by arnest00 — CHANGES_REQUESTED** [review]
Looks great! Looks like you left an extra copy of your self-hosted fonts in the root directory, from [this commit](https://github.com/tisidro/stylewithcss/pull/17/commits/d58cb67c7e1086179036befd27fd576dce8cc387). Delete that and you're good to go.

File: 15custom-fonts/README.md (hunk 1 above)
Comment (tisidro) [reply 15custom-fonts/README.md:23]: Haha, no--I will go in and remove one :)

Comment (tisidro) [reply 15custom-fonts/README.md:23]: Thank you! All updated here: 445225e394360d15666e37b45a25320ee6ef8c18

**Review by arnest00 — APPROVED** [review]
🚀

**Pull request merged**